src/mbyte.c

```diff
@@ -3842,6 +3842,161 @@ utf_head_off(char_u *base, char_u *p)
     return (int)(p - q);
 }
 
+/*
+ * whether space is allowed before/after 'c'
+ * return  TRUE    if not allowed(eat space)
+ *	   FALSE   otherwise
+ */
+    int
+utf_eat_space(cc)
+    int		cc;
+{
+    if ((cc >= 0x2000 && cc <= 0x206F)	      /* General punctuations */
+	    || (cc >= 0x2e00 && cc <= 0x2e7f)   /* Supplemental punctuations */
+	    || (cc >= 0x3000 && cc <= 0x303f)   /* CJK symbols and punctuations */
+	    || (cc >= 0xff00 && cc <= 0xffef))  /* Full width ASCII punctuations */
```

Comment:
U+FF00 to U+FFEF includes not only punctuations but also full width alphabets and half width katakana.

Reply:
I've changed to only include punctuations in the range.